src/roles/index.ts

```diff
@@ -151,24 +176,54 @@ export const permissions = {
       return out;
     });
   },
+  /**
+   * Create a set of permissions specific to any operations involving objects within collections and tenants.
+   *
+   * For all collections, provide the `collection` argument as `'*'`.
+   * For all tenants, provide the `tenant` argument as `'*'`.
+   *
+   * Providing arrays of collections and tenants will create permissions for each combination of collection and tenant.
+   * E.g., `data({ collection: ['A', 'B'], tenant: ['X', 'Y'] })` will create permissions for tenants `X` and `Y` in both collections `A` and `B`.
```

Comment:
suggestion: perhaps this notice would be better placed in to top-most `export permissions {...}` docs, since the idea applies to all permissions which have more than 1 non-action parameter.

Reply:
That's true and I'd agree if there were more than 2 permissions with more than one non-action param. But I think users would be much less likely to read the top-level documentation on this than the docstring directly next to its usage, wdyt?

Reply:
fair point, I was thinking that too